docs/app/pages/components/components-sections/facets/dynamic-facets-ng1/dynamic-facets-ng1.components.html

```diff
@@ -8,6 +8,15 @@
     To use the Dynamic Facet, use the <code>facet-dynamic</code> directive instead of the <code>facet</code> directive. Set
     the
     <code>facet-options</code> attribute with an object containing a <code>title</code> attribute, and an <code>options</code>    property containing collection of objects that follow the interface for the <code>facet-option</code> defined above.
+</p>
+
+<blockquote>
+    <p><strong>Note</strong>: For this type of facet, however, each one of these objects is
+    accessible via the <code>data</code> attribute. Since both <code>name</code> and <code>count</code> are properties within that object,
+    they should be accessed as <code>data.name</code> and <code>data.count</code>, respectively.</p>
```

Comment:
This refers to a `data` attribute. Is this actually an attribute somewhere, or does this refer to the scope variable available to the custom template? If the latter, this piece of information can be relocated to the paragraph mentioning the custom template.

Reply:
This maps to the `data` attribute of the `dynamic-facet-option` directive, but it's not exposed to `dynamic-facet`. So, yes, it will be available as a scope variable in the custom template, but I wanted to place this note here to warn the user that following the `facet-option` interface outlined above won't work in this case.

Reply:
Thanks - it's not very clear, as worded. Could you rewrite the preceding paragraph to provide correct information and remove the note? Also care should be taken to distinguish between HTML attributes and object properties (I appreciate that the original documentation makes this mistake too).